osmaxx-py/excerptconverter/gisexcerptconverter/gis_excerpt_converter.py

```diff
@@ -249,11 +262,13 @@ def execute_task(extraction_order_id, supported_export_formats, execution_config
                     },
                     email=False
                 )
+                logger.exception()
                 raise
             finally:
                 try:
                     subprocess.check_call("docker-compose stop --timeout 0".split(' '))
                     subprocess.check_call("docker-compose rm -v -f".split(' '))
                 except:
+                    logger.exception("couldn't clean up docker containers")
```

Comment:
running `docker ps -a` is a really bad idea, because on a server, you might have hundreds of containers and a huge log for nothing. Furthermore, if this exception occurs, calling something with docker is a bad idea in general, because it might lead to the next exception. That's even more true for the `docker-compose logs`.
What we might add though, is the directory which it was run in - if it didn't get removed, we can look there (but it will be removed if nothing goes terribly wrong since we're using a 'tmp dir' in a with statement).